qt/scientific_interfaces/Indirect/Reduction/ISISEnergyTransferView.cpp

```diff
@@ -146,6 +148,20 @@ void IETView::displayWarning(std::string const &message) {
   QMessageBox::warning(nullptr, "", QString::fromStdString(message));
 }
 
+void IETView::setBackgroundSectionVisible(bool visible) { m_uiForm.gbBackgroundRemoval->setVisible(visible); }
+void IETView::setPlotTimeSectionVisible(bool visible) { m_uiForm.gbPlotTime->setVisible(visible); }
+void IETView::setPlottingOptionsVisible(bool visible) { m_uiForm.fPlottingOptions->setVisible(visible); }
+void IETView::setScaleFactorVisible(bool visible) {
+  m_uiForm.ckScaleMultiplier->setVisible(visible);
+  m_uiForm.spScaleMultiplier->setVisible(visible);
+}
+void IETView::setAclimaxSaveVisible(bool visible) { m_uiForm.ckSaveAclimax->setVisible(visible); }
+void IETView::setNXSPEVisible(bool visible) { m_uiForm.ckSaveNXSPE->setVisible(visible); }
+void IETView::setFoldMultipleFramesVisible(bool visible) { m_uiForm.ckFold->setVisible(visible); }
+void IETView::setOutputInCm1Visible(bool visible) { m_uiForm.ckCm1Units->setVisible(visible); }
+void IETView::setGrouptOutputCheckBoxVisible(bool visible) { m_uiForm.ckGroupOutput->setVisible(visible); }
+void IETView::setGroupOutputDropdownVisible(bool visible) { m_uiForm.cbGroupOutput->setVisible(visible); }
```

Comment:
Sorry, this is picky! Can you leave a space between each of these functions? I think it would help with readability